src/devices/Ft4222/README.md

```diff
@@ -25,6 +25,12 @@ You will need to unzip the file and go to ```LibFT4222-v1.4.2\imports\LibFT4222\
 
 Alternatively, you can register your dll globally. Run from the directory where your ```LibFT4222.dll``` is located the following command in administrator mode: ```regsvr32.exe LibFT4222.dll```
 
+## Linux Requirements
+
+For Linux, you need to download and install the proper version of ```libft4222.so```. The lib include the driver itself in a static binding. You can find the library [here](https://www.ftdichip.com/Support/SoftwareExamples/libft4222-linux-1.4.2.184.tgz). Once downloaded and extracted, run under administrator priviledges ```install4222.sh```. This will install the required library related to your system.
```

Comment:
```suggestion
For Linux, you need to download and install the proper version of ```libft4222.so```. The lib include the driver itself in a static binding. You can find the library [here](https://www.ftdichip.com/Support/SoftwareExamples/libft4222-linux-1.4.2.184.tgz). Once downloaded and extracted, run provided script ```install4222.sh``` as root. This will install the required library related to your system.
```